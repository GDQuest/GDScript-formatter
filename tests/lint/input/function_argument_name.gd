#region Good

func good_function(used_arg, _unused_arg):
	print(used_arg)

func good_typed_function(used_arg: int, _unused_arg: String):
	print(used_arg)

#endregion

#region Bad

func bad_function(badArg, BadArg, _BadUnusedArg):
	print(badArg, BadArg)

func bad_typed_function(badArg: int, BadArg: Array, _BadUnusedArg: String):
	print(badArg, BadArg)

#endregion