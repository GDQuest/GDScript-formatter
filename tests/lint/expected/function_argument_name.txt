tests/lint/input/function_argument_name.gd:13:function-argument-name:error: Function argument 'badArg' should be in snake_case or _private_snake_case format
tests/lint/input/function_argument_name.gd:13:function-argument-name:error: Function argument 'BadArg' should be in snake_case or _private_snake_case format
tests/lint/input/function_argument_name.gd:13:function-argument-name:error: Function argument '_BadUnusedArg' should be in snake_case or _private_snake_case format
tests/lint/input/function_argument_name.gd:16:function-argument-name:error: Function argument 'badArg' should be in snake_case or _private_snake_case format
tests/lint/input/function_argument_name.gd:16:function-argument-name:error: Function argument 'BadArg' should be in snake_case or _private_snake_case format
tests/lint/input/function_argument_name.gd:16:function-argument-name:error: Function argument '_BadUnusedArg' should be in snake_case or _private_snake_case format